frontend/resourceadm/pages/ResourcePage/ResourcePage.tsx

```diff
@@ -8,8 +8,8 @@ import { getResourceDashboardURL, getResourcePageURL } from 'resourceadm/utils/u
 import { DeployResourcePage } from '../DeployResourcePage';
 import {
   useResourceSectorsQuery,
-  useResourceThematicAreaEurovocQuery,
-  useResourceThematicAreaLosQuery,
+  // useResourceThematicAreaEurovocQuery,
+  // useResourceThematicAreaLosQuery,
```

Comment:
what do you think about removing those lines instead? 🤔 😊

Reply:
100%! Thanks